.github/workflows/verify_newproject_script.yml

```diff
@@ -42,4 +42,4 @@ jobs:
 
       - name: Verify generating new project from template
         working-directory: scripts
-        run: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
```

Comment:
Should we add one more step for `template-compose`? 🙏

Reply:
@kaungkhantsoe Would it be better to separate in into two steps? 🙇 

- `Verify generating new project from template-xml`
- `Verify generating new project from template-compose`